pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -259,6 +259,12 @@ exec > >(tee /var/log/user-data.log|logger -t user-data -s 2>/dev/console) 2>&1
 	nodeTaintsArgs := p.getNodeTaintArgs(constraints)
 	kubeletExtraArgs := strings.Trim(strings.Join([]string{nodeLabelArgs, nodeTaintsArgs.String()}, " "), " ")
 
+	if !injection.GetOptions(ctx).AWSENILimitedPodDensity {
+		userData.WriteString(` \
```

Comment:
Do you mind moving towards string concatenation, now that you're in here?

Reply:
Can do. 👍

Reply:
Reminder

Reply:
Ah... now I see why it was a bytes.Buffer instead of string concatenation. The userData has to be base64 encoded, which needs a byte[].

Reply:
Gotcha